### DISCUSSION TITLE
What is "sustain"?

### DISCUSSION THREAD

**Discussion by bauersimon** [zsa/kontroll#13] [Q&A] [answered]
First, thanks a lot for making this API available. Looking forward to build some funky things with it.

Was just wondering what the "sustain" parameter on the RGB and status LED calls are.
Tried changing the value but that had no effect on the keyboard.

**Comment by fdidron** [comment, marked as answer]
Hello,

The sustain parameter allows to specifiy a delay in milliseconds for the rgb / status leds to be set by kontroll. Once the delay expires, the keyboard will go back to its lighting state previous to the kontroll command.

**Reply by bauersimon** [reply]
That is AWESOME! I was actually looking for something like that. Thanks for the clarification.
